x-pack/plugins/security_solution/public/timelines/components/flyout/header/index.tsx

```diff
@@ -39,6 +40,7 @@ const StatefulFlyoutHeader = React.memo<Props>(
     noteIds,
     notesById,
     status,
+    saveTimeline,
```

Comment:
can we move `saveTimeline` action binding to the destination component to avoid unnecessary props drilling?